packages/ui/src/utils/date.js

```diff
@@ -0,0 +1,41 @@
+import dayjs from 'dayjs';
+import chunk from 'lodash.chunk';
+
+/**
+ * @param {Dayjs} [target]
+ * @property {number|string} [params.year]
+ * @property {number|string} [params.month]
+ * @returns {Array<CalendarDay[]>}
+ */
+export const getMonthCalendar = (target) => {
```

Comment:
if method return only last 7 days from month, maybe just rename it to `getLastDaysInMonth`?

Reply:
This is not last 7 days from month, it is a full calendar monts ~4-5 weeks.

Reply:
hmm, i mean, that `getMonthCalendar` name is not clear from external usage